site/operational-guide/concepts.md

```diff
@@ -19,22 +19,23 @@ A peer is a running instance of Weave Net, typically one per host.
 
 ## Peer Name
 
-Peers in the weave network are identified by a 48-bit value formatted
-like an ethernet MAC address e.g. `01:23:45:67:89:ab`. This 'peer
+Peers on the Weave network are identified by a 48-bit value formatted
```

Comment:
Actually, just drop the "on the Weave Network". "Peer" is defined just a couple of lines above, so it is clear here what we are talking about.